lib/drivers/fanaticos-cash.js

```diff
@@ -0,0 +1,50 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/** Fanaticos cash driver. Supports max and
+ * total supply for FCH token
+ *
+ * @memberof Driver
+ * @augments Driver
+ */
+class FanaticosCash extends Driver {
+  constructor(options) {
+    super({
+      supports: {
+        max: true,
+        blockchains: ['Ethereum'],
```

Comment:
Its a bit unclear, but I think they run on their own blockchain
```suggestion
        blockchains: ['Fanatics Cash'],
```